tf_patches/cuda_graph.diff

```diff
@@ -0,0 +1,194 @@
+Revert 8aa20d47ea5c863b9edeb072489ffb7c38b6e8eb
```

Comment:
curious, why do we need to revert this cl?

Reply:
ok Let's follow up in this change later. If I understand correctly this is a compiler issue and it will get fixed if we bump our compiler version? This file will be needed even if get rid of the XRT, so we need to figure out a way before this diff becomes even larger. @stgpetrovic in case you have some insights.

Reply:
Yes, both reverts can be removed once we bump our compiler version.